source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDatePicker.xib

```diff
@@ -14,10 +12,7 @@
         <datePicker contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" datePickerMode="dateAndTime" minuteInterval="1" id="j3j-us-bMu">
             <rect key="frame" x="0.0" y="0.0" width="375" height="215"/>
             <autoresizingMask key="autoresizingMask" widthSizable="YES" flexibleMaxY="YES"/>
-            <color key="backgroundColor" white="0.66666666666666663" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-            <date key="date" timeIntervalSinceReferenceDate="544572879.81824601">
-                <!--2018-04-04 22:14:39 +0000-->
-            </date>
```

Comment:
Out of curiosity, what was this date about?

Reply:
I'm not quite sure, either. Let me check on it.

Reply:
the change is from a topic branch this branch was based on.

Reply:
as i clean up the commits for the merge, i reverted those changes.